src/main/java/org/opentripplanner/raptor/api/request/RaptorRequest.java

```diff
@@ -69,6 +69,15 @@ public SearchParams searchParams() {
     return searchParams;
   }
 
+  /**
+   * Return true is Raptor will use constrained transfers. Raptor will only use constrained
+   * transfers if the {@link RaptorProfile} allowes it and the
+   * {@link SearchParams#constrainedTransfers()} is turned on.
+   */
+  public boolean useConstrainedTransfers() {
+    return profile.spportConstrainedTransfers() && searchParams.constrainedTransfers();
```

Comment:
We probably should force the heuristic to use a profile which supports constrained transfers, if the main search is set to use it.